x-pack/metricbeat/module/mssql/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: mssql
+  title: "mssql"
+  description: >
+    experimental[]
```

Comment:
Should we make this beta directly?

Reply:
I'd prefer to be conservative about this because there are lots of versions and operating systems involved, more than usual I mean because it's just not a Ubuntu or a Centos but also a MS Windows *

Reply:
In general the meaning of Experimental at Elastic is we still might remove the feature in the future, beta means we still make breaking changes. I assume when we get it in I don't see a reason we would remove it again. To mitigate the problems around multiple platforms we should state very clearly what are the platforms we tested against.

Reply:
Replaced experimental mentions with beta. Also the `logp` function